As we are trying to do away with sysvinit remainders,
these are not needed.  All the functionality is present
in our Upstart jobs.
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = intl dbus init util extra conf doc contrib po scripts
+SUBDIRS = intl dbus init util extra doc contrib po scripts
 
 EXTRA_DIST = HACKING
 
